### DISCUSSION TITLE
Presentación de Problemática

### DISCUSSION THREAD

**Discussion by santiemanuel** [santiemanuel/milProgramadoresQA#20] [Simulacro Trabajo Grupal]
# My PyC

La empresa de informática Raid-Zero desea modelizar una computadora de escritorio con todos sus componentes. El analista del equipo planteó la siguiente estructura para representar los componentes físicos de una PC.

## Componentes

![image](https://user-images.githubusercontent.com/3680835/197365827-b979cccb-2a8a-4c8f-9ded-1254de45be18.png)

El arquitecto de software le consultó si tenía más información que aportar sobre el diseño, y la respuesta fue negativa.

Para avanzar al siguiente paso de diseño el CEO asignó de tarea al arquitecto identificar todos los problemas de compatibilidad y limitaciones que puede existir entre los componentes presentados por el analista.

## Tarea T1
La primera tarea de los grupos es realizar un esquema o tabla en donde se listen los potenciales problemas de compatibilidad entre los componentes para el armado correcto de una PyC.

## Modalidad de Presentación
Cada grupo participante en el simulacro debe abrir un hilo nuevo (uno por grupo) con el título `Grupo N_X - My PyC - Tarea T1 - CTY` (reemplazar la X con su número de grupo asignado, y la Y en CTY en la comisión que pertenecen). Una vez publicado se pasará a dar una devolución sobre los problemas encontrados para poder avanzar a la siguiente tarea.

Buena Suerte Equipos

**Comment by Acentor** [comment]
Una duda, se debe trabajar sobre las características de cada componentes? si es asi cuales serian, solo listan las particularidades pero no los datos de las mismas, 
o lo trabajamos en un supuesto cuales pueden ser los limitadores entre componentes, ósea algo como la placa base no soporta la velocidad de la ram o la fuente puede no tener la potencia para la placa de video ?

**Reply by santiemanuel** [reply]
Sería por lo último que dijiste. Podés verlo como que exista un método instalar componente y este tendrá éxito si el componente que agregás es compatible con lo que ya tenés inicialmente, desde el gabinete. Para compatibilidad de RAM en la placa base lo pueden ver desde DDR4/5 para simplificar un poco pero tenés razón en que hay temás de compatibilidad por velocidades en donde el sistema reconoce hasta el máximo que soporta.
